src/engine/renderer/tr_image.cpp

```diff
@@ -2390,8 +2395,13 @@ static void R_CreateContrastRenderFBOImage()
 	tr.contrastRenderFBOImage = R_CreateImage( "_contrastRenderFBO", nullptr, width, height, 1, imageParams );
 }
 
-static void R_CreateBloomRenderFBOImage()
+static void R_CreateBloomRenderFBOImages()
 {
+	if ( !glConfig2.bloom)
+	{
+		return;
+	}
+
```

Comment:
R_InitFBOs does something with these images unconditionally. Does that not cause errors?

Reply:
In this branch, `R_InitFBOs` only does something with it if `glConfig2.bloom` is true (it adds such test).